README.rst

```diff
@@ -95,6 +242,8 @@ Updating a Replication Set
 Dropping a Replication Set
 --------------------------
 
+Replication sets may be dropped via the ``drop`` command.
```

Comment:
o rite

Well there ya go @ivanov.  I'll add a note about this issue to the README.